### ISSUE TITLE
ALTERNATIVE of this plugin

### ISSUE THREAD

**Issue by Absinthze** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#508]
I love this app, and its sad to see all these issues keep piling up.  Abdullah disapeared near the end of 2023.   Hopefully one of you knows how to code and takes over a fork of this repository.

**Comment by zombieyang** [comment]
I started a new one here: https://github.com/zombieyang/sd-ppp

**Comment by Absinthze** [comment]
Thanks for your efforts Zombie, I tested it, but it doesn't have a prompting window and settings for inpainting like the Abdullah version.   The method used in his app is exactly what us arch viz artists need.

**Comment by HallettVisual** [comment]
I'm going to try and use the latest version of ChatGPT to update this for Forge.  Pray for me.

**Comment by ypurpl** [comment]
> I started a new one here: [zombieyang/sd-ppp](https://github.com/zombieyang/sd-ppp?rgh-link-date=2024-08-22T00%3A03%3A08.000Z)

Thanks for this. I would like to know, are there any features in your plugin that's missing from this one?

**Comment by arkijsand** [comment]
> I'm going to try and use the latest version of ChatGPT to update this for Forge. Pray for me.

Hi Matt, did you have any luck with this? :)

**Comment by Absinthze** [comment]
> > I'm going to try and use the latest version of ChatGPT to update this for Forge. Pray for me.
> 
> Hi Matt, did you have any luck with this? :)

No, there are too many files for my lack of coding skills.  I've been using inpainting with Forge.  Works the same.   Its still on my list though!

**Comment by arkijsand** [comment]
> No, there are too many files for my lack of coding skills. I've been using inpainting with Forge. Works the same. Its still on my list though!

Darn, I'm sure we would have seen a post from you about on FB too if you had succeeded. I still use the this old plugin btw and it works for latest PS, but I haven't updated the SD install since 1.7 as I'm afraid it might break something.  

I find the old "realistic vision 5.1 inpaint" model still does a decent job on people inpainting (over 3D people) most of the time, so I still prefer that to having to save output, then upload to webui, then back in PS, having to change something, do it all again. Much easier if kept directly in PS. But I miss the speed and ram optimization from forge. If I had time, I'd try the comfyUI plugin, but it feels like too much of a hassle to setup and troubleshoot.
